### DISCUSSION TITLE
How to force my computer to use GPU to run Stockfish?

### DISCUSSION THREAD

**Discussion by LovelyChess** [official-stockfish/Stockfish#4368] [Q&A] [answered]
Hi, I want to run Stockfish using my dedicated GPU. How do I do this?

**Comment by MinetaS** [comment]
Stockfish doesn't use GPU.

**Comment by ghost** [comment, marked as answer]
https://github.com/official-stockfish/Stockfish/wiki/Stockfish-FAQ#can-stockfish-use-my-gpu-

**Comment by LovelyChess** [comment]
@MinetaS @silversolver1 Thanks. Can you help me to solve issues https://github.com/official-stockfish/Stockfish/issues/4364#issue-1561117070
